src/translation/wininstaller/installerlng.nsi

```diff
@@ -24,16 +24,16 @@
 !include "${ROOT_PATH}\src\translation\wininstaller\es.nsi"
 
 !insertmacro MUI_LANGUAGE "Swedish"
-!include "${ROOT_PATH}\src\translation\wininstaller\se.nsi"
+!include "${ROOT_PATH}\src\translation\wininstaller\sv_SE.nsi"
 
 !insertmacro MUI_LANGUAGE "PortugueseBR"
-!include "${ROOT_PATH}\src\translation\wininstaller\pt_br.nsi"
+!include "${ROOT_PATH}\src\translation\wininstaller\pt_BR.nsi"
 
 !insertmacro MUI_LANGUAGE "Portuguese"
 !include "${ROOT_PATH}\src\translation\wininstaller\pt.nsi"
```

Comment:
pt_PT as we have a separate pt_BR, then?

Reply:
Done